node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -196,16 +136,21 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
     * @return
     */
   def sync(): Future[Unit] = {
+    logger.debug("Started Node.sync")
     val blockchainsF =
       BlockHeaderDAO()(executionContext, chainAppConfig).getBlockchains()
+    val syncPeer = peerManager.peers.head
```

Comment:
IIUC, this means we are always choosing the first peer to sync with (`.head`), is this what we want?

Reply:
Good catch. No, we don't want this. In any case, this `sync` function in `Node` is overriden by `NeutrinoNode` so there's no place where this is used right now.